<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route Sqlite table</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DB:</string>
   </property>
  </widget>
  <widget class="QgsFileWidget" name="fwDBFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From field</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFromField">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teLog">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenDb">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open DB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Geometry field:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGeomField">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>route_geom</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbNoGeom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Geocode only records with no geometry</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHelp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbToField">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To field</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ORS API Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApiKey">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
